### ISSUE TITLE
Locally hosted fonts not working

### ISSUE THREAD

**Issue by briandridge** [remix-run/remix#1921]
### What version of Remix are you using?

1.1.3

### Steps to Reproduce

1. Download this repo: https://github.com/briandridge/remix-architect-fonts/tree/master
2. Run `npm run start` and `npm run dev`
3. Go to localhost:3333
4. See that the Nunito font is not applied to the body, h1, and p elements.

### Expected Behavior

For the elements to appear with the Nunito font.

### Actual Behavior

The elements appear with the system fallback fonts.

I'm not sure if this is the same as https://github.com/remix-run/remix/issues/1153 - but this is using the Architect starter so thought it might be a different problem.

I saw this discussion in Discord, but don't think I'm doing quite the same thing: https://discord.com/channels/770287896669978684/935837819983765524

Thanks so much for all your work, Remix is amazing so far. Apologies in advance if I'm just doing it wrong.

**Comment by MarkusWendorf** [comment]
Hello Brian,

I looked at the repository and noticed that the browser is not even trying to download the font file. So I looked at the css for any obvious errors (ignoring the path for now). Turns out you have an extra `,` in the `src` attribute, which is not correct.

After removing the `,` it tries to download the file but can't find it under `../../fonts/nunito-regular.woff`. 
Relative paths can be error prone, so I would suggest using an absolute path like: `/_static/fonts/nunito-regular.woff`. 

 
```diff
- src: url('../../fonts/nunito-regular.woff'), format('woff');
+ src: url('/_static/fonts/nunito-regular.woff') format('woff');
```  

Reading the config `publicPath: "/_static/build/"` I would have thought that the correct path would be `publicPath + "fonts/nunito-regular.woff"` meaning `"/_static/build/fonts/nunito-regular.woff"` but that is not the case, which is confusing?!
